ui/apps/moderation/src/components/transparency-log/banner.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { useTranslation } from 'react-i18next';
 
 import DS from '@akashaorg/design-system';
-import { ICount } from '@akashaorg/ui-awf-hooks';
+import { ICount } from '@akashaorg/typings/ui';
```

Comment:
count of what?

Reply:
Moderation log items. I'll update the name now